pom.xml

```diff
@@ -200,6 +200,25 @@ THE SOFTWARE.
       <version>1.0.2</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.kohsuke.stapler</groupId>
+      <artifactId>json-lib</artifactId>
```

Comment:
Why? This ought to have been picked up from Jenkins core already.

Reply:
I am trying to fix my `//FIXME response is empty here :(` below, I was thinking that json parser is in this lib so I tried a new version, but it's not working anyway